libindy/src/api/anoncreds.rs

```diff
@@ -386,8 +387,9 @@ pub extern fn indy_prover_create_master_secret(command_handle: i32,
 /// Claim request json.
 ///     {
 ///      "blinded_ms" : <blinded_master_secret>,
-///      "schema_seq_no" : <schema_seq_no>,
-///      "issuer_did" : <issuer_did>
+///      "schema_key" : {name: string, version: string, did: string},
+///      "issuer_did" : string,
+///      "prover_did" : string
```

Comment:
Why do we need prover_did?

Reply:
prover_did is used in Issuer::sign_claim function